drf_sobitie/conf/settings.py

```diff
@@ -90,8 +90,8 @@
     },
 ]
 
-ASGI_APPLICATION = "drf_sobitie.asgi.application"
-WSGI_APPLICATION = "drf_sobitie.wsgi.application"
+ASGI_APPLICATION = "conf.asgi.application"
+WSGI_APPLICATION = "conf.wsgi.application"
```

Comment:
И тут:
```suggestion
ASGI_APPLICATION = "drf_sobitie.conf.asgi.application"
WSGI_APPLICATION = "drf_sobitie.conf.wsgi.application"
```

Тут много где используется подобный путь. Если будет работать моё предложение, добавь его во всем приложении.